plugin-locationcomponent/src/main/res/values/strings.xml

```diff
@@ -4,7 +4,7 @@
     <string name="mapbox_attributionsIconContentDescription">Attribution icon. Activate to show attribution dialog.</string>
     <string name="mapbox_myLocationViewContentDescription">Location View. This shows your location on the map.</string>
     <string name="mapbox_mapActionDescription">Showing a Map created with Mapbox. Scroll by dragging two fingers. Zoom by pinching two fingers.</string>
-    <string name="mapbox_attributionsDialogTitle">Mapbox Maps SDK for Android</string>
```

Comment:
wonder if we should remove the attribution related strings from the locationcomponent, could you try to remove these `mapbox_attribution*` string properties and let the CI run the tests?

Reply:
It seems that tests (mostly) pass without these indeed - the failures in the failed optional tests seems to be unrelated.